prometheus-exporter/exporter.ts

```diff
@@ -163,6 +174,25 @@ class Exporter {
                     return timeInStopWindow >= OVERDUE_THRESHOLD;
                 }).length;
 
+                // Count auto-start schedules that are overdue for start execution
+                const startOverdueCount = schedules.filter(schedule => {
+                    if (!schedule.isInStartWindow) return false;
+                    if (schedule.isClaimable) return false; // Only auto-start schedules
+                    
+                    const timeInStartWindow = now - schedule.schedule.cliffAndFlowDate;
+                    return timeInStartWindow >= OVERDUE_THRESHOLD;
+                }).length;
+
+                // Create incidents based on overdue counts (only if Instatus reporting is enabled)
+                if (ENABLE_INSTATUS_REPORTING) {
+                    // Zero count = healthy, greater than zero = unhealthy
+                    if (startOverdueCount === 0) {
+                        await createIncidentHealthy(processor.networkName, 'vesting_scheduler');
+                    } else {
+                        await createIncidentUnhealthy(processor.networkName, 'vesting_scheduler');
+                    }
+                }
+
```

Comment:
why cover only the start event for vesting schedules, not both like for flow schedules?